var/spack/repos/builtin/packages/kokkos/package.py

```diff
@@ -46,13 +46,33 @@ class Kokkos(Package):
 
     variant('qthreads', default=False, description="enable Qthreads backend")
     variant('cuda', default=False, description="enable Cuda backend")
-    variant('openmp', default=True, description="enable OpenMP backend")
+    variant('openmp', default=False, description="enable OpenMP backend")
+
+    # host architecture variant
+    variant(
+        'host_arch',
+        default='none',
+        values=('none','AMDAVX','ARMv80','ARMv81','ARMv8-ThunderX','Power7','Power8','Power9',
+                'WSM','SNB','HSW','BDW','SKX','KNC','KNL'),
+        description='Set the host architecture to use'
+    )
+
+    # gpu architecture variant
+    variant(
+        'gpu_arch',
+        default='none',
+        values=('none','Kepler30','Kepler32','Kepler35','Kepler37','Maxwell50','Maxwell52',
+                'Maxwell53','Pascal60','Pascal61'),
+        description='Set the GPU architecture to use'
+    )
 
     # Specify that v1.x is required as v2.x has API changes
     depends_on('hwloc@:1')
     depends_on('qthreads', when='+qthreads')
     depends_on('cuda', when='+cuda')
 
+    # Conflicts
+    conflicts()
```

Comment:
What's up with this?

Reply:
An oversight! I will remove it in an update.